app/bingey-api/controllers/title.js

```diff
@@ -1,159 +1,158 @@
+          .custom((value, { req }) => {
+            if (value.length != req.body.seasonCount) {
+              throw new Error('Season episodes mismatch seasons');
+            }
+
+            if (!value.every((ep) => parseInt(ep) != NaN)) {
+              throw new Error('Season episode format incorrect');
+            }
+
+            return true;
+          }),
+        check('latestEpisode.season', 'Latest episode season is required')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .notEmpty(),
+        check('latestEpisode.season', 'Latest episode season format is invalid')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .isInt(),
+        check('latestEpisode.number', 'Latest episode number is required')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .notEmpty(),
+        check('latestEpisode.number', 'Latest episode number format is invalid')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .isInt(),
+        check('latestEpisode.releaseDate', 'Latest episode release date is required')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .notEmpty(),
+        check('latestEpisode.releaseDate', 'Latest episode release date format is invalid')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .isDate(),
+
+        check('upcomingEpisode.season', 'Upcoming episode season is required')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .notEmpty(),
+        check('upcomingEpisode.season', 'Upcoming episode season format is invalid')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .isInt(),
+        check('upcomingEpisode.number', 'Upcoming episode number is required')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .notEmpty(),
+        check('upcomingEpisode.number', 'Upcoming episode number format is invalid')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .isInt(),
+        check('upcomingEpisode.releaseDate', 'Upcoming episode release date is required')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .notEmpty(),
+        check('upcomingEpisode.releaseDate', 'Upcoming episode release date format is invalid')
+          .if(check('isMovie').isIn(['false', 'False', '0']))
+          .isDate(),
+
+        check('plot', 'Plot is required').notEmpty(),
+        check('imageUrl', 'Image URL is required').notEmpty(),
+        check('imageUrl', 'Image URL format is invalid').isURL(),
+        check('similars')
+          .optional()
+          .custom((value) => {
+            if (value && !value.every((id) => id.trim() && id.match(/tt[0-9]+/))) {
+              throw new Error('Similar titles format is invalid');
+            }
+
+            return true;
+          }),
+      ];
+    }
+  }
+};
+
+const parseTitle = (req) => {
+  return new Title({
+    isCustom: true,
```

Comment:
removed _id from here, custom titles will have an id created by mongo, they will have no imdb_id. The field for entering it is also removed from Add title dialog